.env.example

```diff
@@ -97,6 +97,7 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # SLACK_CLIENT_ID= # optional, see plugins/slack_app/README.md
 # SLACK_CLIENT_SECRET= # optional, see plugins/slack_app/README.md
 # SLACK_VERIFICATION_TOKEN= # optional, see plugins/slack_app/README.md
+# SLACK_GLOBAL_BUDDY_REQUEST="<webhook_url>#<channel>" # send notifications for all buddy requests to this channel
```

Comment:
Using `#` as a delimiter might confuse things, since the above line has 3 of them :)

And if they forget to quote the string, they'll only get the first one.